app/views/courses/about.html.erb

```diff
@@ -4,15 +4,34 @@
   <%= govuk_back_link_to(provider_course_path(course.provider.provider_code, course.course_code)) %>
 <% end %>
 
-<%= render partial: 'courses/copy_content_warning',
-                    locals: {
-                      source_course_name: @source_course.name,
-                      source_course_code: @source_course.course_code,
-                      copied_fields: [
-                        ['About the course', '#about_course_wrapper'],
-                        ['Interview process', '#interview_process_wrapper'],
-                        ['How school placements work', '#how_school_placements_work_wrapper']
-                      ]} if params[:copy_from].present? %>
+<% if params[:copy_from].present? %>
+  <% if @source_course.about_course.present? || @source_course.interview_process.present? || @source_course.how_school_placements_work.present? %>
```

Comment:
Just trying to understand the issue being addressed in this change ... is this the key here? That it uses `@source_course.about_course.present?` to decide whether to display this message?

If so, then we should still be able to use the partial and perform the check there. We could pass in `'about_course'`, instead of `'#about_course_wrapper'` for `copied_fields`, and then in the partial:

```
    <% if copied_fields.any? { |_, attr| @source_course.__send__(attr).present? } %>
```

to test whether to print any messages, and use `"##{field[1]}_wrapper"` when passing it to `link_to`.

Reply:
Nice suggestion. Thanks

Reply:
All done. Thanks @misaka